grafana-plugin/src/pages/index.tsx

```diff
@@ -27,24 +27,24 @@ function getPath(name = '') {
 export const pages: { [id: string]: PageDefinition } = [
   {
     icon: 'bell',
-    id: 'incidents',
+    id: 'alert-groups',
     hideFromBreadcrumbs: true,
     text: 'Alert Groups',
     hideTitle: true,
-    path: getPath('incidents'),
+    path: getPath('alert-groups'),
     action: UserActions.AlertGroupsRead,
   },
   {
     icon: 'bell',
-    id: 'incident',
+    id: 'alert-group',
     text: '',
     hideFromTabs: true,
     hideFromBreadcrumbs: true,
     parentItem: {
-      text: 'Incident',
-      url: `${PLUGIN_ROOT}/incidents`,
+      text: 'Alert Group',
+      url: `${PLUGIN_ROOT}/alert-groups`,
     },
-    path: getPath('incident'),
+    path: getPath('alert-groups'),
```

Comment:
This one is plural, the other one was singular. Is this on purpose?

Reply:
Yes it is. We had inconsistency before, and I fixed couple of places to make it all consistent. We have page name like 'alert-groups' in plural and after it will be parameter {id}